Allow individual non-permitted puzzles in multi BLD to count as unsolved. (#670)
thewca/wca-regulations: wca-guidelines.md

- 3j+) [REMINDER] In the past, engraved/embossed parts have been permitted. This is no longer permitted.
- 3j++) [CLARIFICATION] On Clock, pins must not be distinguishable from any other pin of the same side.
- 3j2+) [CLARIFICATION] On Clock, the nine inner clock faces on one side are considered similar pieces.

> - 3kb2+) [CLARIFICATION] Suppose the competitor attempts 10 puzzles and solves 7 successfully. If it is found that two solved puzzles and one unsolved puzzle are not permitted, the competitor is considered to have solved 5 cubes.

Reviewer: >[CLARIFICATION]

Rather [EXAMPLE]?

Author: Done.

Author: Done.

- 3j+) [REMINDER] In the past, engraved/embossed parts have been permitted. This is no longer permitted.
- 3j++) [CLARIFICATION] On Clock, pins must not be distinguishable from any other pin of the same side.
- 3j2+) [CLARIFICATION] On Clock, the nine inner clock faces on one side are considered similar pieces.

> - 3k2+) [ADDITION] The WCA Delegate should not apply a listed exception if they believe the competitor tried to used a non-permitted puzzle on purpose.

Reviewer 2: ```suggestion
- 3k2+) [ADDITION] The WCA Delegate should not apply a listed exception if they believe the competitor tried to use a non-permitted puzzle on purpose.
```

Reviewer 2: This still hasn't been addressed, right?

> - 3kb2+) [EXAMPLE] Suppose the competitor attempts 10 puzzles and solves 8 successfully. If it is found that two solved puzzles and one unsolved puzzle are not permitted, the competitor is considered to have solved 6 puzzles, at the discretion of the Delegate.

Reviewer 2: ```suggestion
- 3kb2+) [EXAMPLE] For example, if a competitor attempts 10 puzzles and solves 8 successfully, and it is found that two solved puzzles and one unsolved puzzle are not permitted, the competitor is considered to have solved 6 puzzles, at the discretion of the WCA Delegate.
```

Consistency.

Reviewer: Btw, is it clear to everybody that the only alternative to considering it as 6 puzzles is to disqualify the whole attempt? Or should we clarify it explicitly?
Could some people interpret "such puzzles may be individually counted as unsolved" as "or they can be counted as solved, at the discretion of the delegate"?

> - 3k2b+) [EXAMPLE] Suppose the competitor attempts 10 puzzles and solves 8 successfully. If it is found that two solved puzzles and one unsolved puzzle are not permitted, the competitor is considered to have solved 6 puzzles, at the discretion of the Delegate.

Reviewer 2: ```suggestion
- 3k2b+) [EXAMPLE] Suppose the competitor attempts 10 puzzles and solves 8 successfully. If it is found that two solved puzzles and one unsolved puzzle are not permitted, the competitor is considered to have solved 6 puzzles, at the discretion of the WCA Delegate.
```

Consistency.
